src/FilaCms.php

```diff
@@ -15,11 +18,39 @@ class FilaCms
 
     protected static $contentModels = null;
 
+    public function systemUser()
+    {
+        $userModel = config('auth.providers.users.model');
+        $system = $userModel::query()->where('email', 'system@filacms')->first();
+        if($system) {
+            return $system;
+        }
+
+        $userFieldsRaw = Schema::getColumnListing((new $userModel())->getTable());
+
+        $excludeFields = [ 'id', 'created_at', 'updated_at', 'deleted_at', 'remember_token', 'email_verified_at', 'password','email'];
+        $data = [
+            'name' => 'System',
+            'email' => 'system@filacms',
+            'password' => Hash::make(Str::random(24))
+        ];
+        $userFields = array_diff($userFieldsRaw, $excludeFields);
+        foreach ($userFields as $key => $field) {
```

Comment:
Is the idea to fill in any custom fields that someone has added to their User Model? What if the column type isn't string?